typing/typemod.ml

```diff
@@ -400,12 +407,15 @@ and approx_sig env ssg =
           let (_path, mty, _od) = type_open env sod in
           approx_sig mty srem
       | Psig_include sincl ->
-          let smty = sincl.pincl_mod in
-          let mty = approx_modtype env smty in
-          let sg = Subst.signature Subst.identity
-                     (extract_sig env smty.pmty_loc mty) in
-          let newenv = Env.add_signature sg env in
-          sg @ approx_sig newenv srem
+          Builtin_attributes.warning_scope sincl.pincl_attributes
```

Comment:
Same thing as above: approx functions should probably not concern themselves with warnings .

Reply:
Thanks, done.